ecommers/lib/ui/pages/authorization/log_in_page.dart

```diff
@@ -11,10 +11,15 @@ import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
 
 class LogInPage extends StatelessWidget {
+  final Function() createANewAccountClicked;
+
+  const LogInPage({this.createANewAccountClicked});
+
   @override
   Widget build(BuildContext context) {
     final localization = I18n.of(context);
     final provider = Provider.of<LogInProviderModel>(context, listen: false);
+    provider.bottomTapCallback = createANewAccountClicked;
```

Comment:
could you please pass that callback in LoginProviderModel constructor?
So you won't be needed to change this file